backend/app/settings.py

```diff
@@ -75,9 +75,11 @@ class Settings(BaseSettings):
     USER_INDEX: str = "user"
 
     TOKEN_URL: str = "/api/v1/token"
+    IS_SSL: bool = True
 
     class Config:
         env_nested_delimiter = "__"
+        env_file = ".env"
```

Comment:
This will override ENV variables provided in docker. The application can be run locally with the `make` command. The application will auto-reload on save.

If we use `docker/.env-local' that should address the running locally without docker if you would like.

Reply:
👍🏻  will update

Reply:
@KentonParton  how can this work for the local development without docker?

Reply:
Can we not specify the `docker/.env-local` in `.vscode/launch.json` envFile?